src/eth/EthereumCdpService.js

```diff
@@ -105,9 +105,16 @@ export default class EthereumCdpService extends PrivateService {
     });
   }
 
-  async lockEth(cdpId, amount, unit = ETH) {
-    await this._conversionService().convertEthToWeth(amount, unit);
-    return this.lockWeth(cdpId, amount);
+  lockEth(cdpId, amount, unit = ETH) {
+    const promise = (async () => {
+      await 0;
```

Comment:
this is a horrible wart -- if it's omitted, the value of `promise` two lines after is undefined... hoping to find some way around this